core/spec.md

```diff
@@ -2017,7 +2130,7 @@ xRegistry-latestId: STRING
 xRegistry-latestUrl: URL
 xRegistry-description: STRING ?
 xRegistry-documentation: STRING ?
-xRegistry-tags: STRING (JSON map) ?
+xRegistry-labels-KEY: STRING *
```

Comment:
Same comment as earlier (either ignore the suggestion in both places, or change both)

Reply:
Looks like this happens in other places as well...

Reply:
tweaked them all